sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/NewApiTokenController.java

```diff
@@ -6,22 +6,25 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import com.mercedesbenz.sechub.webui.configuration.SecHubAccessService;
+
 @Controller
 public class NewApiTokenController {
     @Autowired
     NewApiTokenService newApiTokenService;
 
     @Autowired
-    SecHubServerAccessService accessService;
+    SecHubAccessService accessService;
 
     @GetMapping("/requestNewApiToken")
     String requestNewApiToken(Model model) {
-        // TODO: Replace with real user email once proper authentication is added
-        String email = "email@example.org";
+        // TODO Jeremias Eppler, 2023-09-27: Replace with real user email once proper
+        // authentication is added
+        String emailAddress = "email@example.org";
 
-        newApiTokenService.requestNewApiToken(email);
-        model.addAttribute("sechubServerUrl", accessService.getSecHubServerUrl());
-        model.addAttribute("userEmail", email);
+        model.addAttribute("sechubServerUrl", accessService.getSecHubServerUri());
```

Comment:
Please introduce (a class with) constants for the model and use the constants everywhere. Means we would have central place and maintenance would be easier.